hack/jenkins/common.sh

```diff
@@ -96,6 +96,12 @@ for stale_dir in "${TEST_ROOT}"/*; do
   export KUBECONFIG="${stale_dir}/kubeconfig"
 
   if [[ -d "${MINIKUBE_HOME}" ]]; then
+    if [[ -f "${MINIKUBE_HOME/tunnels.json" ]]; then
+      echo "Stale tunnels.json:"
```

Comment:
why is it "stale"?

Reply:
Because jobs should be cleaning up after themselves at the end of a run: https://github.com/kubernetes/minikube/blob/master/hack/jenkins/common.sh#L188

This path should only be executed when jobs that are aborted so that post-run cleanup isn't executed.